src/components/PasswordSetterBox.js

```diff
@@ -36,6 +36,12 @@ class PasswordSetterBox extends Nimiq.Observable {
         }
 
         this._onInputChangeValidity(false);
+
+        window.onpopstate = /** @param {PopStateEvent} ev */ ev => {
+            if (ev.state && !ev.state.repeat) {
```

Comment:
Additionally, I think the reset event should only fire if the property is explicitly `false` and not also for `undefined`.